sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/meta/provider/avro/AvroTableProvider.java

```diff
@@ -38,15 +42,10 @@
  * LOCATION '/tmp/persons.avro'
  * }</pre>
  */
-@AutoService(TableProvider.class)
-public class AvroTableProvider extends InMemoryMetaTableProvider {
-  @Override
-  public String getTableType() {
-    return "avro";
-  }
-
-  @Override
-  public BeamSqlTable buildBeamSqlTable(Table table) {
-    return new AvroTable(table.getName(), table.getSchema(), table.getLocation());
+@Internal
+@Experimental
+public class AvroTableProvider extends SchemaCapableIOTableProviderWrapper {
```

Comment:
```suggestion
@AutoService(TableProvider.class)
public class AvroTableProvider extends SchemaCapableIOTableProviderWrapper {
```

We can keep these annotations the same as they were, since this class should work exactly the same as it used to from the user perspective.